showcase/src/app/app.component.ts

```diff
@@ -31,7 +31,7 @@ import { APP_MENU_CONFIG } from "./app-menu.config";
  * Top Level Component
  */
 @Component({
-	selector: "app",
+	selector: "app-showcase",
```

Comment:
Is this change required ?

Reply:
the prefix must be followed by something.